koku/db_performance/test_db_performane.py

```diff
@@ -0,0 +1,110 @@
+#
```

Comment:
This filename should be `test_db_performance.py` instead of `test_db_performane.py`

Reply:
This should be addressed.